wiki/Contests/Official_support/en.md

```diff
@@ -65,7 +65,7 @@ Contests which abide by the following criteria are eligible for official support
   - The final score must be calculated as follows: `judge X's final score = (judge X's raw score - judge X's average raw score) / judge X's standard deviation`.
 - **The contest's host must moderate their judges' comments.** Comments which are either offensive, belittling, low effort, or otherwise inappropriate may be removed at the committee's discretion.
 - **The contest's results must be publicly displayed with judge scores/comments visible.** The judges may be anonymised publicly, but the osu! team may ask for un-anonymised judge details if necessary.
-- **The contest must have at least 4 judges.** Contests with fewer judges allow each judge too much influence on overall results.
+- **The contest must have at least 4 judges.** Contests with fewer judges allow each judge too much influence on overall results. This also applies to contests which incorporate a screening phase prior to judging.
```

Comment:
wording feels pretty ehh here, this reads as we're also requiring 4 judges for contests which incorporate a screening phase prior to judging, which doesn't make sense as that's already the case.

I'd want us to use something that explicitly requires 4 or more screeners, i.e.

```suggestion
- **The contest must have at least 4 judges.** Contests with fewer judges allow each judge too much influence on overall results.
- **contests which incorporate a screening phase prior to judging must also have at least 4 screeners.**
```

Reply:
I agree. Was probably too stubborn not making a new bulletpoint altogether. An extra one won't hurt, will apply.